TomTomSDKExamples/Examples/UseCase/OnlineNavigationContent.swift

```diff
@@ -51,13 +53,14 @@ final class NavigationController: ObservableObject {
         let routeReplanner = RouteReplannerFactory.create(routePlanner: routePlanner, replanningPolicy: .findBetter)
         let locationProvider = DefaultCLLocationProvider()
         let simulatedLocationProvider = SimulatedLocationProvider(delay: .tt.seconds(1))
-        let navigationConfiguration = NavigationConfiguration(
-            apiKey: Keys.apiKey,
-            locationProvider: simulatedLocationProvider,
-            routeReplanner: routeReplanner,
-            betterProposalAcceptanceMode: .automatic
-        )
-        let navigation = Navigation(configuration: navigationConfiguration)
+        let navigationConfiguration = OnlineTomTomNavigationFactory.Configuration(
+                    locationProvider: simulatedLocationProvider,
+                    routeReplanner: routeReplanner,
+                    apiKey: Keys.apiKey,
+                    betterProposalAcceptanceMode: .automatic
+                )
+         
+        let navigation = try! OnlineTomTomNavigationFactory.create(configuration: navigationConfiguration) as! Navigation
```

Comment:
Using optional binding and error handling would be a nicer approach than force unwrapping.

Reply:
Agree with @muhammedtanriverdi-tomtom, we need to keep all good practices as we do in the GoSDK production code.

Reply:
Issue will be addressed here: https://jira.tomtomgroup.com/browse/GOSDK-14640

Reply:
As discussed with @YuryLapitsky-TomTom and @muhammedtanriverdi-tomtom the forced try has been replaced with the introduction of `fatalError`